building_energy_standards_data/database_tables/hvac_minimum_requirement_unitary_air_conditioners.py

```diff
@@ -7,102 +7,110 @@
 RECORD_HELP = """
 Must provide a tuple that contains:
 template: TEXT
+equipment_type: TEXT
 cooling_type: TEXT
 heating_type: TEXT
 subcategory: TEXT
+application: TEXT
+electric_power_phase: NUMERIC
+region: TEXT
 minimum_capacity: NUMERIC
 maximum_capacity: NUMERIC
 start_date: TEXT
 end_date: TEXT
 minimum_seasonal_efficiency: NUMERIC
+minimum_seasonal_efficiency_2: NUMERIC
```

Comment:
I think we should probably correct this. SEER stands for Seasonal Energy Efficiency Ratio, not seasonal efficiency.